src/screens/TopicMemberScreen/index.js

```diff
@@ -88,7 +88,7 @@ const TopicMemberScreen = () => {
     return () => {
       scrollY.removeListener(listener);
     };
-  }, [scrollY]);
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dependency array for the `useEffect` hook has been changed from `[scrollY]` to `[]`. This means that the effect will only run once after the initial render, and not on subsequent re-renders when `scrollY` changes. If this is not the intended behavior, consider reverting the change.
```diff
-  }, []);
+  }, [scrollY]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->